hibernate-core/src/main/java/org/hibernate/engine/transaction/jta/platform/internal/JBossStandAloneJtaPlatform.java

```diff
@@ -27,6 +27,7 @@ public class JBossStandAloneJtaPlatform extends AbstractJtaPlatform {
 	private static final WildFlyStandAloneJtaPlatform wildflyBasedAlternative = new WildFlyStandAloneJtaPlatform();
 
 	@Override
+	@SuppressWarnings("nullness")
```

Comment:
Use the helper class approach we talked about to reduce code duplication and handle nullness in that helper correctly.